meson_options.txt

```diff
@@ -7,6 +7,7 @@ option('blob', type: 'boolean', value: false, description: 'Compile just one bin
 option('subprojects_check', type: 'boolean', value: true, description: 'Check if git subprojects are up-to-date. Might be useful to disable this when developing on a different subproject version')
 option('portable', type: 'boolean', value: false, description: 'Make rizin installation moveable, by using relative paths instead of absolute ones')
 option('extra_prefix', type: 'string', value: '', description: 'Extra load path prefix (absolute path) for plugins, sdb, sigdb, etc.')
+option('n_thread_limit', type: 'integer', min: 1, value: 32767)
```

Comment:
Why so big? Could have been less, I think.

Reply:
because on linux is high. on windows is low. also there are many servers using arm having 1k cores.